src/Microsoft.AspNet.Server.Kestrel/Filter/SocketInputStream.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using IllyriadGames.ByteArrayExtensions;
```

Comment:
But you know we're not going to have a dependency on  `"IllyriadGames.ByteArrayExtensions": "1.0.0-beta-10"` in our web server right? :smile: